Rakefile

```diff
@@ -134,7 +134,9 @@ case @osfamily
 end
 
 # The Puppet 4 load path
-@p4libdir = "/opt/puppetlabs/puppet/lib/ruby/vendor_ruby"
+@p4home = "/opt/puppetlabs/puppet"
```

Comment:
There are two bindirs
1. Public (for our bins e.g. puppet, facter) /opt/puppetlabs/bin
2. Private (for vendored deps e.g. ruby) /opt/puppetlabs/puppet/bin

So, suggested values for these would be

    @p4home = "/opt/puppetlabs"
    @p4bindir = "#{@p4home}/bin"
    @p4puppetdir = "#{@p4home}/puppet"
    @p4libdir = ""#{@p4puppetdir}/lib/ruby/vendor_ruby"


@joshcooper does this look sane to you?